extensions/amp-base-carousel/0.1/carousel.js

```diff
@@ -665,6 +665,11 @@ export class Carousel {
       this.scrollContainer_.setAttribute('horizontal', this.axis_ === Axis.X);
       this.scrollContainer_.setAttribute('loop', this.isLooping());
       this.scrollContainer_.setAttribute('snap', this.snap_);
+
+      // Explicitly set loop since it we ultimately decide if it should loop
+      // and some CSS styling relies on amp-base-carousel[loop=...]
+      this.element_.setAttribute('loop', this.isLooping());
```

Comment:
Does this overwrite the user provided `loop` value? What about for responsive attribute formats, like `loop="(min-width: 1000px) true, false)"`?

Reply:
Unneeded.